python/lsst/validate/drp/matchreduce.py

```diff
@@ -163,7 +161,7 @@ def build_matched_dataset(repo, dataIds, matchRadius=None, safeSnr=50.,
     blob.magKey = blob._matchedCatalog.schema.find("base_PsfFlux_mag").key
     # Reduce catalogs into summary statistics.
     # These are the serialiable attributes of this class.
-    reduceSources(blob, blob._matchedCatalog, safeSnr)
+    summarizeSources(blob, filterSources(blob._matchedCatalog, brightSnr=safeSnr))
```

Comment:
Because you were trying to avoid the confusing name `safeSnr`, why not replace it here (all 3 instances in "build_matched_dataset") with `brightSnrMin`?

Reply:
Good call. I was going to leave it to avoid having to change any configs until I remembered that it doesn't actually get used anywhere in the code (until DM-22138 gets fixed).